nitter/lang/de/strings.php

```diff
@@ -8,3 +8,4 @@ function string_plural_select_de($n){
 ;
 $a->strings["Nitter server"] = "Nitter Server";
 $a->strings["Save Settings"] = "Einstellungen Speichern";
+$a->strings["In an attempt to protect your privacy, links to Twitter in this posting were replaced by links to the Nitter instance at %s"] = "Um deine Privatsphäre zu schützen, wurden in diesem Beitrag Links nach ";
```

Comment:
This looks incomplete

Reply:
You are right, all the split lines need to be removed :-/